README.md

```diff
@@ -70,6 +73,11 @@ helm delete caretta --namespace caretta
 ```
 Note that if persistent storage was enabled in the installation, it may not be deleted automatically by this command.
 
+## Requirements
+Linux kernel version on all K8s nodes running Caretta must be newer than 4.16. Currently Caretta supports only x64 systems.
```

Comment:
I'd structure this as a checklist:
Requirements
- Linux kernel version >= 4.16
- x64 system 
- [CO:RE](url) support, supported Linux distributions can be found [here](url)